### ISSUE TITLE
Zul'Gurub Edge of Madness, Brazier of Madness missing.

### ISSUE THREAD

**Issue by Phenomenal101** [dalaranwow/dalaran-wow#5938]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Enter Zul'Gurub and run to the Edge of Madness boss area, at the time that I was there it showed 2 bosses on the tablets as active and the other two as "May return one day" as supposed to. However the Brazier that is meant to be extinquished with "Gurubashi Mojo Madness" is missing.
2. Brazier is meant to be standing on the top of the stairs leading to the gong and tablets. As can be seen at the start of this YouTube video: https://www.youtube.com/watch?v=_BymsuB9VG4&t=9s
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

- https://www.wowhead.com/spell=24264/extinguish
- https://wowgaming.altervista.org/aowow/?object=180327

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

If instead or if in the meantime the items and trinkets obtainable from this encounter could be added to the Vote Store that would be a fix as well.

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report. This issue has been fixed and a random boss will properly spawn upon using the Brazier of Madness now. This fix will be active after the next scheduled maintenance.

**Comment by Phenomenal101** [comment]
Hello Loth,

Has the maintenance been yet because I am in Zul'Gurub right now and
there's no Brazier. There was only 1 boss that was mentioned to be "At the
edge of madness" right now so that's working correctly.


Kind Regards,

 Phenomenal.

Op ma 8 mrt. 2021 om 18:07 schreef Lothloryen <notifications@github.com>:

**Comment by Lothloryen** [comment]
Hello there,

Yes, this commit was missing after the fire incidence. You can try it again now, it should be working.

Thanks for the heads up.

**Comment by Phenomenal101** [comment]
No problem, thank you for taking the time to fix it.

It works, doing it right now.

Op do 11 mrt. 2021 om 15:06 schreef Lothloryen ***@***.***>:
